src/language/providers/completionItemCache.ts

```diff
@@ -0,0 +1,17 @@
+import { CompletionItem } from "vscode";
+import LRU from "lru-cache";
```

Comment:
Where does this package come from? `lru-cache` is on npm, but there's no change to the `package.json` file in this PR.

Reply:
Just ran `npm ls lru-cache` and saw it got installed via another package - awesome. Nonetheless, we still need it as an explicit `dependency` requirement in the `package.json` file.